virttest/qemu_vm.py

```diff
@@ -548,17 +548,21 @@ def process_sandbox(devices, action):
                 return sandbox_option
             if action == "add":
                 sandbox_option = " -sandbox on"
+                _params = {
+                    "elevateprivileges": params.get(
+                        "qemu_sandbox_elevateprivileges", "deny"
+                    ),
+                    "obsolete": params.get("qemu_sandbox_obsolete", "deny"),
+                    "resourcecontrol": params.get(
+                        "qemu_sandbox_resourcecontrol", "deny"
+                    ),
+                    "spawn": params.get("qemu_sandbox_spawn", "deny"),
+                }
+                for opt, val in _params.items():
+                    if val is not None:
```

Comment:
it looks val is never 'None'

Reply:
Good catch! Just simply delete this conditional judgement. Thank you!